src/main/java/com/CezaryZal/manager/filters/validator/FormValidator.java

```diff
@@ -9,8 +10,12 @@
 
     private static final int MIN_LENGTH = 3;
 
-    boolean isEmpty(AuthenticationRequest authenticationRequest) {
-        return authenticationRequest == null;
+    boolean isNull(FormUser formUser) {
+        return formUser == null;
+    }
+
+    boolean isNull(String arg){
+        return arg == null;
```

Comment:
Logika tych metod mówi że to raczej powinno być setNull niż isNull

Reply:
SetNull?
Tu nic nie przesyłamy, tylko sprawdzamy, czemu set?

Reply:
Racja, nie zauważyłem że są dwa "="